README.md

```diff
@@ -125,3 +125,20 @@ Configuring Shibboleth
 * visit '/Shibboleth.sso/Metadata' to download and review the metadata, rename to a filename of your preference
 * upload the SP metadata to http://testshib.org/register.html or your Identiy Provider (IdP)
 * login by clicking on 'Sign into Shibboleth' and choosing one of the available identities
+
+Audit Fix
+---
+* To fix data with triple files:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'triples','sample-sparql.txt'```
+* To remove irrelevant theses dates:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'thesis'```
+* To fix reindex:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'reindex'```
+A set of rake tasks is also added for index jobs:
+rake hydranorth:solr:index[id]         Index a single object with ID
+rake hydranorth:solr:index_pairtree[input]   Index with a pairtree structure
+rake hydranorth:solr:update_generic_file_index  Index all Generic File objects
+
+/bin/fix/fix.rb is to update all the namespace uris
```

Comment:
Seems to expect bin/fix/fix.rb to be in the current directory. Am I invoking this wrong?

```
[root@hydranorthdev hydranorth]# ./bin/fix/run.sh 
ruby: No such file or directory -- fix.rb (LoadError)
...
```